prep_bm_host.sh

```diff
@@ -230,6 +230,16 @@ ifup "$BM_BRIDGE"
 ifdown "$BM_INTF"
 ifup "$BM_INTF"
 
+###----------------------------------------------------------------###
+### Configure local registry to allow for disconnected deployments ###
+###----------------------------------------------------------------###
+
+if [[ "$DISCONNECTED_INSTALL" =~ True|true|yes ]]; then
+(
+    ./scripts/gen_local_registry.sh
```

Comment:
Because this script creates/extracts the openshift-install binary, we want to check for `$DISCONNECTED_INSTALL` later in this prep_bm_host script to prevent it from overwriting the binary.  See the section labeled "Prepare Openshift binaries".  We would want to wrap this in the disconnected install conditional check:

```cd /tmp

        # Always download and overwrite binaries to ensure 
        # proper versions
        #
        curl -O "$OCP_INSTALL_BINARY_URL"
        tar xvf "${OCP_INSTALL_BINARY_URL##*/}"
        sudo mv openshift-install "$REQUIREMENTS_DIR"
```

Reply:
Makes total sense. Thanks for your comments.